### PR TITLE
Bug 1141479 - Fix string for default activities

### PR DESCRIPTION
**Pull request by fcampo**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95310a274cef)

File: shared/locales/activities/activities.en-US.properties [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-
 # DIAL
-default-activity-makecall     = Make call
+default-activity-dialnumber     = Make call
 
 # NEW
-default-activity-opencontact  = Open contact
-default-activity-sendmail     = Send email
-default-activity-sendmessage  = Send message
+default-activity-newcontact     = Open contact
```
Comment (flodolo) [inline shared/locales/activities/activities.en-US.properties]: Is this correct? From a key called newcontact I'd expect something like "Create contact"

File: shared/locales/activities/activities.en-US.properties [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
-default-activity-openpdf      = Open PDF file
-default-activity-openvideo    = Open video
+default-activity-viewurl        = Open link
+default-activity-viewurl-email  = Send email
+default-activity-viewpdf        = Open PDF file
+default-activity-viewvideo      = Open video
 
 # UPDATE
-default-activity-opencontact  = Open contact
+default-activity-updatecontact  = Open contact
```
Comment (flodolo) [inline shared/locales/activities/activities.en-US.properties]: Same here: "Update contact" or "Open contact"?

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=34b0fc8aba40)

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3b46e86f49ee)

File: shared/locales/activities/activities.en-US.properties [hunk 3]
```diff
@@ -1,31 +1,50 @@
 # Default activities label
 
-# PICK
-default-activity-pickimage    = Take picture
+# LOCALIZATION NOTE This strings are used to describe actions available for
```
Comment (flodolo) [inline shared/locales/activities/activities.en-US.properties]: "These strings are used to describe available actions that can be associated by default to an application" ?

This -> singular, these -> plural (also in the rest of the comment)

File: shared/locales/activities/activities.en-US.properties [hunk 4]
```diff
@@ -1,31 +1,50 @@
...
+#   which an application can be associated for default launch.
+#   e.g. "Open link" -> Firefox
+#        "Open image"-> Gallery
+#
+# Some separated actions are grouped as one for this purposes, so when an app
```
Comment (flodolo) [inline shared/locales/activities/activities.en-US.properties]: I'm not sure that the information from this point on is really useful to localizers. I think "developer-relevant" comments should be in the code, not in localization files.

File: shared/locales/activities/activities.en-US.properties [hunk 5]
```diff
@@ -1,31 +1,50 @@
...
 
 # DIAL
-default-activity-makecall     = Make call
+default-activity-dialnumber     = Make call
 
 # NEW
-default-activity-opencontact  = Open contact
-default-activity-sendmail     = Send email
-default-activity-sendmessage  = Send message
+# default-activity-newcontact same as default-activity-opencontact
```
Comment (flodolo) [inline shared/locales/activities/activities.en-US.properties]: Is this comment still relevant?

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=91a68a6e87b5)

**Pull request merged**